MODULE.bazel

```diff
@@ -14,11 +14,11 @@ bazel_dep(
 ### JavaScript ecosystem ###
 ############################
 
-bazel_dep(name = "aspect_rules_js", version = "2.1.2")
-bazel_dep(name = "aspect_rules_ts", version = "3.3.2")
+bazel_dep(name = "aspect_rules_js", version = "2.5.0")
+bazel_dep(name = "aspect_rules_ts", version = "3.7.0")
```

Comment:
How do you find the "working combination"? Can one read it from somewhere or is it just trial and error?

Reply:
Mostly bumping to newest version and reverting change if something breaks. Trial and error.